ortools/base/raw_logging.cc

```diff
@@ -25,13 +25,17 @@
 #include "ortools/base/logging.h"
 #include "ortools/base/logging_utilities.h"
 
-#if !defined(_MSC_VER)
+#if !defined(_MSC_VER) && !defined(__EMSCRIPTEN__)
 #include <sys/syscall.h>  // for syscall()
 #define safe_write(fd, s, len) syscall(SYS_write, fd, s, len)
 #else
+#if !defined(__EMSCRIPTEN__)
```

Comment:
Can you reverse the test

#if defined(__EMSCRIPTEN__)

#else

Reply:
Have changed the test.